archive/frames/management/commands/purgeolderframes.py

```diff
@@ -0,0 +1,25 @@
+from django.core.management.base import BaseCommand
```

Comment:
I usually like to do standard lib imports first, then 3rd party lib imports, then local project imports:
```
from datetime import timedelta
import logging

from django.core.management.base import BaseCommand
from django.utils import timezone

from archive.frames.models import Frame
```